### PR TITLE
[WIP] Update readme

### PR DESCRIPTION
**Pull request by ZombineDev**

### CODE REVIEW

**Comment by ZombineDev** [comment]
I also need to update dub.json and the code.dlang.org references in the readme, but I'm not sure what to do about them. Seems like we need a shared dlang-community account.

**Comment by BBasile** [comment]
https://code.dlang.org/packages/dyaml will continue to work normally.

**Comment by ZombineDev** [comment]
Don't we need to manually press the "Trigger manual update" button, to make the registry fetch the new tags?

**Comment by BBasile** [comment]
When you hit this button all the packages are updated. So we just have to wait that someone updates its package and dyaml will be processed implicitly.

File: dub.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "dyaml-dlang-tour",
+    "name": "dyaml",
     "description": "YAML parser and emitter",
     "authors": [ "Ferdinand Majerech" ],
     "libs": [],
```
Comment (BBasile) [inline dub.json]: line 9 there's an address to change

Comment (ZombineDev) [inline dub.json]: tnx will fix

Comment (BBasile) [inline dub.json]: And there are more links to change in the readme too

Comment (ZombineDev) [inline dub.json]: should be fixed now

File: README.rst [hunk 2]
```diff
@@ -80,9 +77,9 @@ Installing and tutorial
 -----------------------
 
 See the
-`Getting Started <https://github.com/kiith-sa/D-YAML/wiki/Getting-Started>`_
+`Getting Started <https://github.com/dlang-community/D-YAML/wiki/Getting-Started>`_
 tutorial and other tutorials that can be found at the
-`GitHub wiki <https://github.com/kiith-sa/D-YAML/wiki>`_
+`GitHub wiki <https://github.com/dlang-community/D-YAML/wiki>`_
 and in the ``doc/html/tutorials/`` directory in the package.
 For advanced usage, see the `documentation <http://defenestrate.eu/docs/dyaml/>`_.
```
Comment (BBasile) [inline README.rst]: Got to remove this sentence.

File: README.rst [hunk 3]
```diff
@@ -2,30 +2,27 @@
...
-(`almost <https://github.com/kiith-sa/D-YAML/wiki/Differences-between-D:YAML-and-the-YAML-specification>`_)
+(`almost <https://github.com/dlang-community/D-YAML/wiki/Differences-between-D:YAML-and-the-YAML-specification>`_)
 compliant to the YAML 1.1 specification. D:YAML is based on
 `PyYAML <http://www.pyyaml.org>`_ created by Kirill Simonov. 
 
 D:YAML is designed to be easy to use while supporting the full feature set of
 YAML. To start using it in your project, see the
-`Getting Started <https://github.com/kiith-sa/D-YAML/wiki/Getting-Started>`_
+`Getting Started <https://github.com/dlang-community/D-YAML/wiki/Getting-Started>`_
 tutorial and the `documentation <http://defenestrate.eu/docs/dyaml/>`_.
```
Comment (BBasile) [inline README.rst]: finish the sentence after `tutorial` and remove the invalid link.

**Comment by ZombineDev** [comment]
> When you hit this button all the packages are updated. So we just have to wait that someone updates its package and dyaml will be processed implicitly.

Great, than we should good now. We may need a dlang-community account in the future, but I guess we'll cross that bridge when we get there.

**Comment by BBasile** [comment]
No there's the 2 links to http://defenestrate.eu to remove !

**Comment by ZombineDev** [comment]
> No there's the 2 links to http://defenestrate.eu to remove !

I didn't saw your comments before I posted my message. The dead links are removed now.

**Comment by BBasile** [comment]
Oops v0.6.1 of course

**Comment by ZombineDev** [comment]
Alright, will do shortly.

**Comment by ZombineDev** [comment]
> Oops v0.6.1 of course

done

**Comment by BBasile** [comment]
Thanks for handling tis quickly. We're good now. All repos are under control and up-to-date :ok_hand:

**Comment by ZombineDev** [comment]
No problem, glad I could help. BTW, I'm still not sure if the package will be updated automatically. See:
1) https://github.com/dlang/dub-registry/blob/7b779b5b2244546a2c145e491d7949fccfa6468a/views/my_packages.package.dt#L26

2) https://github.com/dlang/dub-registry/blob/48dc9a4ad05144b4516bc5b2a002906f8e3458a4/source/dubregistry/web.d#L485

3) https://github.com/dlang/dub-registry/blob/923714b9046ce1274dfcd1d5773e04bdd6c1b5cc/source/dubregistry/registry.d#L73 and https://github.com/dlang/dub-registry/blob/923714b9046ce1274dfcd1d5773e04bdd6c1b5cc/source/dubregistry/registry.d#L400

As far as I understand, only an authenticated user may trigger a package update.
@s-ludwig, can you confirm?

**Comment by BBasile** [comment]
It's is updated now :+1: , look at code.dlang.org

**Comment by ZombineDev** [comment]
Great, thanks @BBasile for the news!

**Comment by s-ludwig** [comment]
@ZombineDev: Currently all packages get polled about every 30 minutes: https://github.com/dlang/dub-registry/blob/923714b9046ce1274dfcd1d5773e04bdd6c1b5cc/source/app.d#L35 (hopefully someone will be able to change this to immediate updates using push notifications at some point in the not-so-distant future).

**Pull request merged**